coverage-reports/generate_report.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+# Specify snapshot version
+SNAPSHOT_VERSION_TAG="0.8.0-SNAPSHOT"
+
+# Step 1: Change to the project directory
+echo "Changing to the project directory..."
+cd ../sdk || exit 1
+
+# Step 2: Run 'mvn clean install' in the project directory
+echo "Running 'mvn clean install' in the project directory..."
+mvn clean install
+
+# Step 3: Execute run_sc_tests.sh
+echo "Executing script for integration tests..."
+cd ../qa || exit 1
+./run_sc_tests.sh -jacoco
+
+# Step 4: Generate the JaCoCo code coverage report
+echo "Generating JaCoCo code coverage report..."
+
+JACOCO_JAR_PATH="$HOME/.m2/repository/org/jacoco/org.jacoco.cli/0.8.9/org.jacoco.cli-0.8.9-nodeps.jar"
+EXEC_PATH="$SIDECHAIN_SDK/coverage-reports/sidechains-sdk-${SNAPSHOT_VERSION_TAG}/sidechains-sdk-${SNAPSHOT_VERSION_TAG}-jacoco-report.exec"
```

Comment:
I would add a check to make sure the `SIDECHAIN_SDK` env var is not empty at the beginning of the script

Reply:
done